test/jdk/java/awt/print/PrinterJob/JobName/PrinterJobName.java

```diff
@@ -21,173 +21,54 @@
  * questions.
  */
 
-/**
+import java.awt.Graphics;
+import java.awt.Graphics2D;
+import java.awt.print.PageFormat;
+import java.awt.print.Printable;
+import java.awt.print.PrinterJob;
+
+/*
  * @test
  * @bug 4205601
  * @summary setJobName should be used by PrinterJob
  * @key printer
+ * @library /java/awt/regtesthelpers
+ * @build PassFailJFrame
  * @run main/manual PrinterJobName
  */
-
-import java.awt.*;
-import java.awt.print.*;
-
 public class PrinterJobName implements Printable {
+    private static final String THE_NAME = "Testing the Jobname setting";
```

Comment:
```suggestion
    private static final String THE_NAME = "Testing the Job name setting";
```
Spell it with space.